ga4gh/_protocol_definitions.py

```diff
@@ -207,6 +207,52 @@ def __init__(self, **kwargs):
         """
 
 
+class Attributes(ProtocolElement):
+    """
+    Type defining a collection of attributes associated with various
+    protocol   records.  Each attribute is a name that maps to an
+    array of one or more   values.  Values can be strings, external
+    identifiers, or ontology terms.   Values should be split into the
```

Comment:
I created a low-priority issue to track it; it's not something you introduced https://github.com/ga4gh/server/issues/993